### ISSUE TITLE
How to get the performance of multi-task learning at 68.8 on CIFAR100?

### ISSUE THREAD

**Issue by HaitaoWen** [imirzadeh/CL-LinearModeConnectivity#1]
Hi @imirzadeh!
I am trying to reproduce the performance of multi-task learning in the paper of [MC-SGD](https://arxiv.org/pdf/2010.04495.pdf). To carry out this experiment, i concatenate all data of 20 tasks together, and each sample is a triplet (x, y, t). The task identifier t is used to select corresponding classifer head to calculate the cross entropy loss. In this way and use the model code in the repository [Stable SGD](https://github.com/imirzadeh/stable-continual-learning), i can get 64.3 on CIFAR100 which is close to 64.8 reported in the paper of Stable SGD. However, when i use the model code in this repository, i can only get 57.2 on CIFAR100 which is significantly lower than 68.8 reported in MC-SGD. I want to know if there is something wrong in my implementation. Hyper parameters are as follows:

> optimizer: SGD
> learning rate: 0.1
> weight decay: 0
> momentum: 0.7
> batch size: 10
> epochs: 1
